bin/nrnpyenv.sh.in

```diff
@@ -60,7 +60,7 @@ fi
 # The first argument passed to this script is the value of nrniv -pyexe
 pyexe_arg="$1"
 
-WHICH=which
+WHICH="command -v"
```

Comment:
This works for me. e.g.
```
$ which python
/home/hines/.pyenv/shims/python
$ command -v python
/home/hines/.pyenv/shims/python
```
But I'm kind of curious about whether the following points to a potential issue.
```
$ which ls
/usr/bin/ls
$ command -v ls
alias ls='ls --color=auto'
```

Reply:
Fedora does not ship `which` by default:

```plaintext
$ command -v which; echo $?
1
```

I'm sure it's part of some core builtins package, but I've no idea what pulls it in as a dependency. I'm guessing there are other distros that don't ship it by default either. One alternative is to first check if `which` exists (via `command -v which`), and if not, default to `WHICH="command -v"`. Would that work?

Reply:
Perhaps it is the case that whatever is produced by ```command -v``` is actually better than ```which```.  I do notice that
```
$ type -P ls
/usr/bin/ls
```
is more similar to ```which```.  Does ```type``` work on fedora?